backend/app/testchat/test_gm.py

```
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
```

Ruff (0.8.2)

Shebang is present but file is not executable

(EXE001)